commerce-account-service/src/main/java/com/liferay/commerce/account/service/persistence/impl/CommerceAccountGroupFinderImpl.java

```diff
@@ -41,6 +42,10 @@
 	public List<Long> findAccountUserIdsFromAccountGroupIds(
 		long[] commerceAccountGroupIds, int start, int end) {
 
+		if (ArrayUtil.isEmpty(commerceAccountGroupIds)) {
+			return new ArrayList<>();
```

Comment:
Updated and sent a pull to change thanks. Is the rule ArrayList for initializing but Collections for returning?

Reply:
`ArrayList` is like the default implementation for a `List` that you actually want to add stuff to. `Collections.emptyList` is a singleton -- an unmodifiable, empty list. So if you don't have any elements to return, it makes sense to reuse the existing singleton object rather than allocating a new object just to say "nothing here". Using this consistently also helps with readability, as `Collections.emptyList` becomes the "nullish" list return object.